src/Adapter/Adapter.php

```diff
@@ -35,9 +34,29 @@ public function create(Stream $stream): void;
      */
     public function appendTo(StreamName $streamName, Iterator $domainEvents): void;
 
-    public function load(StreamName $streamName, int $minVersion = null): ?Stream;
-
-    public function loadEvents(StreamName $streamName, array $metadata = [], int $minVersion = null): Iterator;
-
-    public function replay(StreamName $streamName, DateTimeInterface $since = null, array $metadata = []): Iterator;
+    public function load(
+        StreamName $streamName,
+        int $fromNumber = 0,
```

Comment:
Is there a chance to get an event with version '0', or should it '1'?
Perhaps it is just irrelevant and '0' could stay here. Not sure..